### DISCUSSION TITLE
error (65): floating invalid.

### DISCUSSION THREAD

**Discussion by MJiaHao** [Anura3D/Anura3D_OpenSource#40] [General]
I have changed some parameters in the Tutorial in Chapter 6 Column collapse.I defined all component material types as saturated material -- fully coupled, and selected rigid bodies as the base material model entities. But I encountered some contact propertise errors in my calculations. forrtl: error (65): floating invalid. I tried many methods, but didn't get an effective solution.
![error](https://user-images.githubusercontent.com/116955023/198830664-6ff52627-d1d1-4d51-8059-10c96eabec6a.png)

**Comment by murphy-john** [comment]
This is a Fortran error usually caused when a value gets assigned to infinity (often dividing by zero). That's often caused when dividing something by zero. You'll have to use the debugging feature in Visual Studio and look for where the issue occurs. 

There are also some issues with combining the rigid bodies and different material types.

**Reply by MJiaHao** [reply]
Thank you very much for your advice. I think I have found the cause of the problem. I prescribed the wrong constraint displacement direction of the rigid body material. Thank you for your reply.

**Reply by Shachen75** [reply]
Hi, MJiaHao, can you provide more details on how you solved this problem?  I have encountered the same warning message when I modelled the base material as rigid body. The constraint conditions were also modified but seemed not working.  Thanks.

**Comment by TJU-CZT** [comment]
I got the same error as you when I selected 2-phase, double-point formulations.I checked the rigid body displacement without errors.
